tutorials/hsdk-ngc-container/.vscode/workspace.code-workspace

```diff
@@ -0,0 +1,11 @@
+{
+	"folders": [
+		{
+			"path": ".."
+		},
+		{
+			"path": "/home/holoscan/examples"
+		}
+	],
+	"settings": {}
+}
```

Comment:
Please add a blank line here.

Reply:
I've removed all files inside `.vscode/` directory since we are copying it from the NGC container.